packages/keystone/apolloErrorFormatter.js

```diff
@@ -96,130 +97,222 @@ function _patchKnownErrorCases (error, result) {
     }
 }
 
-function _extractInnerGQLError (error) {
-    const innerErrors = get(error, 'errors', [])
-    return innerErrors.find(err => get(err, 'name') === 'GQLError')
+class NonError extends Error {
+    constructor (message) {
+        super(util.inspect(message))
+        this.name = 'NonError'
+        Error.captureStackTrace(this, NonError)
+    }
 }
 
-/**
- * Use it if you need to safely prepare error for logging or ApolloServer result
- * @param {Error} error -- any error
- * @param {Boolean} hideInternals -- do you need to hide some internal error fields
- * @param {Boolean} applyPatches -- do you need to apply a common error message patches
- * @returns {import('graphql').GraphQLFormattedError}
- */
-const safeFormatError = (error, hideInternals = false, applyPatches = true) => {
-    const result = {}
+function _ensureError (error) {
```

Comment:
What is this util for...

By looking in constructor of non-error it should be string, but here it's called error....It error or message or what? JSDocs is appreciated

Reply:
I just want to drop the requirements for https://www.npmjs.com/package/ensure-error util. And hardcode it inside with the tests.

You can check it here: https://github.com/sindresorhus/ensure-error/blob/main/index.js

I'll add some docs for that.

Reply:
Check DOMA-10342